02_bag.ipynb

```diff
@@ -320,7 +320,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "js.filter(lambda record: record['name'] == 'Alice').pluck('transactions').take(3)"
+    "(js.filter(lambda record: record['name'] == 'Alice')\n",
+    "   .pluck('transactions')\n",
+    "   .take(3))"
```

Comment:
The next few cells all use method chaining. Doing the same here for consistency and readability.

![image](https://user-images.githubusercontent.com/17503044/100467554-a53bea80-30a0-11eb-9b64-daa4de653289.png)